app/assets/javascripts/facility_directory.es6

```diff
@@ -0,0 +1,198 @@
+    }
+
+    function toggle_by_practices_created(){
+        let newUrl = addSortParams() + "practices_created";
+        window.location.href = newUrl;
+    }
+
+    function toggle_by_adoptions(){
+        let newUrl = addSortParams() + "adoptions";
+        window.location.href = newUrl;
+    }
+
+    function addSortParams(){
+        let curUrl = window.location.href;
+        let paramAsc = getParameterByName("asc", newUrl)
+        let newUrl = removeParam("asc", curUrl);
+        newUrl = removeParam("sortby", newUrl);
+        newUrl.includes("?") ? newUrl += "&" : newUrl += "?";
+        newUrl += "asc=";
+        if(paramAsc == null){
+            newUrl += "false";
+        }
+        else{
+            paramAsc == "true" ? newUrl += "false" : newUrl += "true";
+        }
+        newUrl += "&sortby=";
+        return newUrl;
+    }
+
+
+
+    function removeParam(key, sourceURL) {
+        let rtn = sourceURL.split("?")[0],
+            param,
+            params_arr = [],
+            queryString = (sourceURL.indexOf("?") !== -1) ? sourceURL.split("?")[1] : "";
+        if (queryString !== "") {
+            params_arr = queryString.split("&");
+            for (var i = params_arr.length - 1; i >= 0; i -= 1) {
+                param = params_arr[i].split("=")[0];
+                if (param === key) {
+                    params_arr.splice(i, 1);
+                }
+            }
+            if (params_arr.length) rtn = rtn + "?" + params_arr.join("&");
+        }
+        return rtn;
+    }
+
+    function getParameterByName(name, url = window.location.href) {
+        name = name.replace(/[\[\]]/g, '\\$&');
+        let regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+            results = regex.exec(url);
+        if (!results) return null;
+        if (!results[2]) return '';
+        return decodeURIComponent(results[2].replace(/\+/g, ' '));
+    }
+
+    function loadMoreRecords(){
+        let btn = document.getElementById("btn_facility_directory_load_more");
+        let numRecs = parseInt(btn.getAttribute("num_recs"));
+        numRecs += 20;
+        let curUrl = window.location.href;
+        let paramMore = getParameterByName("more", newUrl)
+        let newUrl = removeParam("more", curUrl);
+        newUrl.includes("?") ? newUrl += "&" : newUrl += "?";
+        newUrl += `more=${numRecs}`;
+        window.location.href = newUrl;
+    }
+
+    function loadMorePractices(){
+        debugger
+        let btn = document.getElementById("btn_facility_show_page_load_more");
+        let numPracticeRecs = parseInt(btn.getAttribute("num_practice_recs"));
+        numPracticeRecs += 3;
+        let curUrl = window.location.href;
+        let paramMore = getParameterByName("more", newUrl)
+        let newUrl = removeParam("more", curUrl);
+        newUrl.includes("?") ? newUrl += "&" : newUrl += "?";
+        newUrl += `practices=${numPracticeRecs}`;
+        window.location.href = newUrl;
+    }
+
+});
+
```

Comment:
Can we utilize the `showMoreItems` library that we have as opposed to refreshing the page in order to load the next 20 items?